examples/alerts/new_github_stars.yml

```diff
@@ -0,0 +1,35 @@
+# Alert when there are new GitHub Stars utilizing keepstate
+alert:
+  id: new-github-stars
+  description: Get new GitHub Stars
+  steps:
+    - name: get-github-stars
+      provider:
+        type: github.stars
+        config: " {{ providers.github-pat }}"
+        with:
+          repository: "keephq/keep"
+          previous_stars_count: " {{ state.new-github-stars.-1.alert_context.alert_steps_context.get-github-stars.results.stars }} " # state.alert-id.-1 for last run
```

Comment:
I guess we should find a better way to do it since this is not indicative.

Maybe something like - 
```
          previous_stars_count: " {{ state.previous_stars_count }} "
```
And add the option to keep the state with a state block
```
state: 
  - name: previous_stars_count
     value: steps.get-github-stars.results.new_stargazers
```

Reply:
We can think about simplifying the "syntax" although it's pretty straightforward, I'll think about a way to extend it and will open an issue